empress/recon_vis/tanglegram.py

```diff
@@ -0,0 +1,141 @@
+"""
+tanglegram.py
+Visualizes tanglegrams using matplotlib
+Berto Garcia, Sonia Sehra
+"""
+
+from empress.recon_vis import utils, plot_tools, tree
+
+VERTICAL_OFFSET = 20
+HORIZONTAL_SPACING = 10
+LEAF_SPACING = 5
+HOST_COUNTER = 0
+PARASITE_COUNTER = 0
+
+def render(host_dict: dict, parasite_dict: dict, phi: dict, show_internal_labels: bool) -> None:
+    """
+    Render tanglegram
+    :param host_dict - host tree (dictionary representation)
+    :param parasite_dict - parasite tree (dictionary representation)
+    :param phi - tip mapping dictionary
+    :param show_internal_labels - boolean indicator of whether internal node names should
+        be displayed
+    :return FigureWrapper object 
+    """
+    fig = plot_tools.FigureWrapper("Host | Parasite")
```

Comment:
Should the `HOST_COUNTER` and `PARASITE_COUNTER` resets every time we call render?